digdag-standards/src/main/java/io/digdag/standards/operator/PyOperatorFactory.java

```diff
@@ -101,59 +104,130 @@ public TaskResult runTask()
                 .build();
         }
 
-        private Config runCode(Config params)
+        private Config runCode()
+                throws IOException, InterruptedException
+        {
+            final Config params = request.getConfig()
+                    .mergeDefault(request.getConfig().getNestedOrGetEmpty("py"));
+            final Config stateParams = TaskState.of(request).params();
+            final Path projectPath = workspace.getProjectPath(); // absolute
+
+            final CommandStatus status;
+            if (!stateParams.has("commandStatus")) {
+                // Run the code since command state doesn't exist
+                status = runCode(params, projectPath);
+            }
+            else {
+                // Check the status of the code running
+                final ObjectNode previousStatusJson = stateParams.get("commandStatus", ObjectNode.class);
+                status = checkCodeState(params, projectPath, previousStatusJson);
+            }
+
+            if (status.isFinished()) {
+                final int statusCode = status.getStatusCode();
+                if (statusCode != 0) {
+                    // Remove the polling state after fetching the result so that the result fetch can be retried
+                    // without resubmitting the code.
+                    stateParams.remove("commandStatus");
+                    throw new RuntimeException("Python command failed with code " + statusCode);
+                }
+
+                final Path workspacePath = workspace.getPath();
+                final Path outputPath = workspacePath.resolve(status.getIoDirectory()).resolve("output.json");
```

Comment:
CommandStatus.getIoDirectory is relative path from CommandExecutorContent.getLocalProjectPath. So this should use `CommandExecutorContent.getLocalProjectPath().resolve(status.getIoDirectory())` instead of resolving from workspacePath.

Reply:
Fixed it on https://github.com/treasure-data/digdag/pull/835/commits/e52acbb29ca33db8180079bc5713833a3cf9c868#diff-d92680253d5ee8051d8ae4b4a0374904R137. Thank you.